### PR TITLE
Add more to database backup task

### PR DESCRIPTION
**Pull request by Ian-Nicholls**
**WHAT**

Added missing variables needed to be passed to the script (all of them!)

**WHY**

It needs them to work

### CODE REVIEW

File: govwifi-api/database-backup-task.tf [hunk 1]
```diff
@@ -218,7 +263,7 @@ resource "aws_cloudwatch_event_target" "backup-rds-to-s3" {
   "containerOverrides": [
     {
       "name": "database-backup",
-      "command": ["bundle", "exec", "rake", "database_backup.sh[${var.rack-env}]"]
+      "command": ["pwd; ls -l; ./database_backup.sh"]
```
Comment (sarahseewhy) [inline govwifi-api/database-backup-task.tf:266]: More for my curiosity than anything else, why change the command?

**Review by sarahseewhy — APPROVED** [review]
LGTM, just that question about the command change but I don't think it's blocking.

File: govwifi-api/database-backup-task.tf (hunk 1 above)
Comment (Ian-Nicholls) [reply govwifi-api/database-backup-task.tf:266]: well bundle is:
bundle. bundle - Ruby Dependency Management bundle COMMAND [--no-color] [--verbose] [ARGS]. Description. Bundler manages an application's 
and rake is:
Rake is a tool you can use with Ruby projects. It allows you to use ruby code to define "tasks" that can be run in the command line. Rake can be downloaded and included in ruby projects as a ruby gem. Once installed, you define tasks in a file named "Rakefile" that you add to your project.30 Nov 2013

(I googled!) so both are for running Ruby - this is a bash script though!

Comment (sarahseewhy) [reply govwifi-api/database-backup-task.tf:266]: Ah, I just needed that last bit about converting the command to a bash script. I wanted to know why we moved from a rake task to bash.

**Pull request merged**